rust/osm2lanes/src/road/lane.rs

```diff
@@ -12,14 +12,16 @@ pub enum Lane {
         // TODO, we could make this non-optional, but remove the field for designated=foot?
         direction: Option<LaneDirection>,
         designated: LaneDesignated,
+        width: Option<Metre>,
```

Comment:
So what's the overall picture -- if we can scrape lane width from OSM, this is filled out, but otherwise the caller can use `Locale::travel_width`?

A thing to think about for the near future: in OSM, usually width of the entire road is mapped (and of course it's a bit complicated what "entire road" actually covers). We may need a way to stash that, and to think about algorithms that can divide the total width across lanes in some reasonable way (probably by guessing a default width by lane type, adding that up, taking a ratio of that sum to the total in OSM, and scaling appropriately)